src/DayPicker.tsx

```diff
@@ -362,7 +362,7 @@ export function DayPicker(props: DayPickerProps) {
                           value={dateLib.getMonth(calendarMonth.date)}
                         />
                       ) : (
-                        <span role="status" aria-live="polite">
+                        <span>
```

Comment:
We should remove these elements, as they've been added only for a the live region. Thoughts?

Reply:
They are still used to display text when the layout is not purely `dropdown`, like `dropdown-months` and `dropdown-years`:

<img width="365" alt="image" src="https://github.com/user-attachments/assets/e80b94c3-fcb1-407e-acda-669f6f33c27f" />

So they are still needed to display the calendar information for users without screen reader.

Reply:
I meant to remove the `span`, not its content -  but out of the scope of this PR!